Gemfile

```diff
@@ -4,3 +4,5 @@ source "https://rubygems.org"
 
 gem "octokit"
 gem "rss"
+gem "asciidoctor", "~> 2.0.0"
```

Comment:
# @****

Reply:
I could leave the dependencies in the `Gemfile` and only remove those that are no longer needed.

However, the dependencies get added back with the commits introducing the code that needs them. I want it this way because that makes it harder to leave unneeded dependencies in the `Gemfile` by mistake.

Reply:
I guess we could just reword the commit message on caf86c4
to mention why we're dropping them all and adding some back in later.

Reply:
Yep, I reworded the commit message:  032d3b70e